lib/main.dart

```diff
@@ -27,6 +31,13 @@ void main() async {
   SystemChrome.setPreferredOrientations([DeviceOrientation.portraitUp]);
   MediaKit.ensureInitialized();
   await vod.init();
+  databaseFactory = databaseFactorySqflitePlugin;
+  fallbackDatabase = await MatrixSdkDatabase.init(
+    '${AppConfig.applicationName}-${DateTime.now().millisecondsSinceEpoch}',
+    database: !PlatformInfos.isWeb
+        ? await openDatabase('./${AppConfig.applicationName}.db')
+        : null,
+  );
```

Comment:
How to make smooth migration for our user, no need to logging out, still use the application?